test/parserTest.js

```diff
@@ -1593,27 +1593,74 @@ test("Using a helper in dynamic attribute and @html", async(t) => {
 	t.is(html, `<template key="otherBlue">testBlue</template>`);
 });
 
+test("Using a helper (without this) in dynamic attribute and @html", async(t) => {
+	let component = new WebC();
+	component.setHelper("helper", (a) => { return a+"Blue"; });
+	component.setContent(`<template :key="helper('other')" @html="helper('test')"></template>`);
+
+	let { html } = await component.compile();
+
+	t.is(html, `<template key="otherBlue">testBlue</template>`);
+});
```

Comment:
**Suggestion**: I like the added error handling and the tests for it, but I think this test could be expanded with a few more cases test some different use cases like this one:

```js
test("Using props without “this”", async t => {
	let component = new WebC();

	component.setInputPath("./test/stubs/props-no-this.webc");

	let { html, css, js, components } = await component.compile({
		data: {
			variable: "value",
			3: "number key",
			nested: {
				object: {
					fn: () => 'Boo!',
				},
			},
		}
	});

	t.deepEqual(js, []);
	t.deepEqual(css, []);
	t.deepEqual(components, [
		"./test/stubs/props-no-this.webc",
	]);

	t.is(html, `<p key="value"></p>
<p key="value"></p>
<p key="2"></p>
<p key="number key"></p>
<p key="test"></p>
<p key="Boo!"></p>`);
});
```

**test/stubs/props-no-this.webc**:

```
<p :key="variable"></p>
<p :key="this.variable"></p>
<p :key="2"></p>
<p :key="3"></p>
<p :key="process.env.NODE_ENV"></p>
<p :key="this.nested.object.fn()"></p>
```